splunk/packages/edit-table/src/EditTable.jsx

```diff
@@ -113,7 +124,7 @@ const EditTable = (props) => {
         function convertLabelObjectToKey(obj) {
             const newObj = {};
             Object.keys(obj).forEach((label) => {
-                const key = labelMap.find((m) => m.label === label)?.key || label;
+                const key = labelMap[label];
```

Comment:
maybe that's better to read?
```suggestion
                const key = label2key[label];
```